Microsoft.DotNet.Interactive/Events/PackageAdded.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Interactive.Commands;
+
+namespace Microsoft.DotNet.Interactive.Events
+{
+    public class PackageAdded : KernelEventBase
+    {
+        public PackageAdded(AddPackage addPackage): base(addPackage)
+        {
+            if (addPackage == null)
+            {
+                throw new System.ArgumentNullException(nameof(addPackage));
+            }
+
+            PackageReference = addPackage.PackageReference;
```

Comment:
It's preferable to take in the `PackageReference` directly here. Then long as the parameter name matches the property name (case-insensitive), the deserialization will work correctly.